apps/user-profile/src/components/modules/UserOnboardingModal/RoleCard/RoleCard.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { RoleCardContainer, PicWrapper, TextContents, IconImg} from './StyledRoleCard'
+
+// @description This component renders the RoleCard Component.
```

Comment:
you can comment in multiple lines using `/** @description  ....comment */`

Reply:
Completed.